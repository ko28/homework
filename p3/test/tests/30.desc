Create and remove 5 aliases in a queue order
