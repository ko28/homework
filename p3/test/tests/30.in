alias ls /bin/ls
alias whoami /usr/bin/whoami
alias ll /bin/ls -l
alias psh /usr/bin/ps --help
alias la /bin/ls -l -a
alias
unalias ls
unalias whoami
unalias ll
alias
unalias psh
unalias la
alias
exit
