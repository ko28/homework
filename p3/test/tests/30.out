alias ls /bin/ls
alias whoami /usr/bin/whoami
alias ll /bin/ls -l
alias psh /usr/bin/ps --help
alias la /bin/ls -l -a
alias
ls /bin/ls
whoami /usr/bin/whoami
ll /bin/ls -l
psh /usr/bin/ps --help
la /bin/ls -l -a
unalias ls
unalias whoami
unalias ll
alias
psh /usr/bin/ps --help
la /bin/ls -l -a
unalias psh
unalias la
alias
exit
